app/src/components/fields/SingleDateField.tsx

```diff
@@ -40,6 +46,9 @@ const SingleDateField: React.FC<IDateProps> = (props) => {
   return (
     <LocalizationProvider dateAdapter={AdapterMoment}>
       <DatePicker
+        slots={{
+          openPickerIcon: CalendarIcon
```

Comment:
Ill try, I feel like maybe there was some warning about inlining functions though, because react complains about it re-rendering every time (If I am remembering correctly).

Reply:
Right, yeah I realized that after commenting. No worries if it can't be done

Reply:
Well, In-lined them, and it seems to be happy

Reply:
Wait spoke too soon, sonarqube is complaining about it

Reply:
Ahh too bad 😪